tests/core/test_cli_entry.py

```diff
@@ -27,9 +27,15 @@ def tearDown(self):
         if path.exists(status_file_path):
             os.remove(status_file_path)
 
+        # Clean up the logger in the case where a test fails to complete
+        print("Cleaning up directory logger")
+        if logger.directory_logger:
+            logger.remove_directory_logger()
+
+    @patch("core.cli_entry.progress")
     @patch("core.cli_entry.api_handler")
     @patch("core.cli_entry.parsing_handler")
-    def test_valid_all_functions_called(self, mock_parsing_handler, mock_api_handler):
+    def test_valid_all_functions_called(self, mock_parsing_handler, mock_api_handler, mock_progress):
```

Comment:
We now have to mock the `progress` module, because the `parsing_handler` no longer checks for new.